cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBot.Build.cs

```diff
@@ -10,7 +10,7 @@ public UrdfBot(ReadOnlyTargetRules target) : base(target)
     {
         SP_LOG_CURRENT_FUNCTION();
 
-        PublicDependencyModuleNames.AddRange(new string[] {"CoreUtils"});
+        PublicDependencyModuleNames.AddRange(new string[] {"CoreUtils", "XmlParser"});
```

Comment:
Does this need to be here or can it be moved to `CommonModuleRules` without breaking other plugins?